### DISCUSSION TITLE
add default keyboard shortcut to toggle "Access as Unique Name"

### DISCUSSION THREAD

**Discussion by lighttroupe** [godotengine/godot-proposals#11461] [Editor]
Perhaps **Shift-5** or **Ctrl-Shift-5**

**Comment by Meorge** [comment]
I think I remember seeing a similar proposal recently, where someone also suggested you could type a % as the first character in a node's name and it would make it a unique node. Both a keyboard shortcut and the node-name approach could be quite handy!

**Comment by sockeye-d** [comment]
You can already set one in the shortcuts menu in the editor settings

**Comment by lighttroupe** [comment]
Good point @sockeye-d thanks!  

It seems that it allows Shift-5 but then it doesn't actually work.  Ctrl-Shift-5 _does_ work.  (Godot 4.2.2 on Linux)

So I guess this issue is about adding one as default.

**Comment by lighttroupe** [comment]
@Meorge Yeah [that one](https://github.com/godotengine/godot-proposals/issues/11386) pairs nicely with this one.

**Comment by Mickeon** [comment]
I am converting this to a discussion as you did not fill the proposal's form properly.

**Reply by lighttroupe** [reply]
OK thanks.

I removed the boilerplate because the reason for a keyboard shortcut is always the same: it's faster than grabbing the mouse.
